kura/org.eclipse.kura.wire.component.provider/src/main/java/org/eclipse/kura/internal/wire/subscriber/CloudSubscriberOptions.java

```diff
@@ -66,8 +66,7 @@
     int getSubscribingQos() {
         int subscribingQos = DEFAULT_QOS;
         final Object qos = this.properties.get(CONF_QOS);
```

Comment:
I have noticed that also. Previously it was misplaced, and the null check was performed after get on the map is called. It is due to extracting the piece of code to local variable using Eclipse feature and I forgot to add the null check thereafter. My proposal would be to add `requireNonNull` in the constructor which would save the verbosity of null checks in all the getters.